front/app/components/IdeasMap/IdeaMapCard.tsx

```diff
@@ -141,21 +142,31 @@ const FooterValue = styled.div`
 
 interface Props {
   ideaMarker: IIdeaMarkerData;
-  isPBIdea: boolean;
   onClose?: () => void;
   className?: string;
   projectId: string;
+  phaseId?: string;
 }
 
 const IdeaMapCard = memo<Props>(
-  ({ ideaMarker, isPBIdea, onClose, className, projectId }) => {
+  ({ ideaMarker, onClose, className, projectId, phaseId }) => {
     const tenant = useAppConfiguration();
+    const phase = usePhase(phaseId || null);
     const project = useProject({ projectId });
     const { windowWidth } = useWindowSize();
     const smallerThanMaxTablet = windowWidth <= viewportWidths.largeTablet;
 
     const [hovered, setHovered] = useState(false);
 
+    const isPBProject =
```

Comment:
I don't immediately understand what PB project means - maybe a more descriptive name would be helpful?